### PR TITLE
fetch data to movies and chip

### PR DESCRIPTION
**Pull request by Ahmedoudouh**

### CODE REVIEW

File: src/api/api-chip.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React, { useEffect } from "react";
```
Comment (younessoudouh) [inline src/api/api-chip.js:1]: what's that?

File: src/Componnets/chip/chip.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { TiDelete } from "react-icons/ti";
 
 import "./style-button.css";
 
-export const Chip = ({ name }) => {
+export const Chip = ({ name,id }) => {
```
Comment (faouzioudouh) [inline src/Componnets/chip/chip.jsx]: space before `id`

File: src/api/api-chip.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+export const ChipApi = ({ setTodos }) => {
+
```
Comment (faouzioudouh) [inline src/api/api-chip.js]: remove empty line.

File: src/api/api-chip.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React, { useEffect } from "react";
+
+export const ChipApi = ({ setTodos }) => {
```
Comment (faouzioudouh) [inline src/api/api-chip.js]: is this a custom hook?

File: src/pages/movies/movies.jsx [hunk 5]
```diff
@@ -1,31 +1,63 @@
...
+import React, { useState } from "react";
 
 import { Chip } from "../../Componnets/chip/chip";
+import { MovieCard } from "../../Componnets/movie-card/MovieCard";
+import { useMoviesApi } from "../../api/api-movies";
+import { useChipApi } from "../../api/api-chip";
+import { Pagination } from "../../Componnets/pagination/pagination";
 
 export const Movies = () => {
+
```
Comment (faouzioudouh) [inline src/pages/movies/movies.jsx:10]: remove empty line

File: src/pages/movies/movies.jsx [hunk 6]
```diff
@@ -1,31 +1,63 @@
...
+  const [currentPage, setPage] = useState(1);
+  const { data, isLoading, isError } = useMoviesApi(currentPage);
+  const { data:dataChip } = useChipApi();
```
Comment (faouzioudouh) [inline src/pages/movies/movies.jsx:13]: space after `data`

Comment (faouzioudouh) [inline src/pages/movies/movies.jsx:13]: space after `data`

**Review by faouzioudouh — APPROVED** [review]

**Pull request merged**